docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -3,36 +3,18 @@ title: Use Teleport for Application Access
 description: How to configure Teleport for Application Access.
 ---
 
-# Connecting Web Applications
+## Prequisites
 
-Download the latest version of Teleport for your platform from our [downloads page](https://goteleport.com/teleport/download)
-and follow the installation [instructions](../../installation.mdx).
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-## Start Auth/Proxy service
+(!docs/pages/includes/tctl.mdx!)
 
-Create a configuration file for a Teleport service that will be running
-auth and proxy servers:
+- A host where you will run the Teleport Application Service.
 
-```yaml
-teleport:
-  data_dir: /var/lib/teleport
-auth_service:
-  enabled: "yes"
-proxy_service:
-  enabled: "yes"
-  # Set public address proxy will be reachable at.
-  public_addr: teleport.example.com:3080
-ssh_service:
-  enabled: "no"
-```
-
-(!docs/pages/includes/permission-warning.mdx!)
-
-Start the service:
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx) or try our Teleport Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
```

Comment:
```suggestion
If you have not yet tried Teleport, we recommend that you begin our Teleport Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
```
The `edition-prereqs-tabs.mdx` partial includes links to Getting Started guides, so I think we can cut this to just the link to the learning track.